orc8r/cloud/go/services/streamer/providers/providers.go

```diff
@@ -33,34 +35,79 @@ func (p *MconfigProvider) GetStreamName() string {
 }
 
 func (p *MconfigProvider) GetUpdates(gatewayId string, extraArgs *any.Any) ([]*protos.DataUpdate, error) {
-	resp, err := configurator.GetMconfigFor(gatewayId)
+	res, err := configurator.GetMconfigFor(gatewayId)
 	if err != nil {
-		return nil, errors.Wrap(err, "error getting mconfig from configurator")
+		return nil, errors.Wrap(err, "get mconfig from configurator")
 	}
 
-	if extraArgs != nil {
-		// Currently, only use of extraArgs is mconfig hash
-		receivedDigest := &protos.GatewayConfigsDigest{}
-		if err := ptypes.UnmarshalAny(extraArgs, receivedDigest); err == nil {
-			glog.V(2).Infof("Received, generated config digests: %v, %v\n",
-				receivedDigest,
-				resp.Configs.Metadata.Digest.Md5HexDigest,
-			)
-			return mconfigToUpdate(resp.Configs, resp.LogicalID, receivedDigest.Md5HexDigest)
-		}
+	// TODO(T71525030): add back below support for mconfig digests
+	//if extraArgs != nil {
+	//	// Currently, only use of extraArgs is mconfig hash
+	//	receivedDigest := &protos.GatewayConfigsDigest{}
+	//	if err := ptypes.UnmarshalAny(extraArgs, receivedDigest); err == nil {
+	//		glog.V(2).Infof(
+	//			"Received, generated config digests: %v, %v\n",
+	//			receivedDigest,
+	//			res.Configs.Metadata.Digest.Md5HexDigest,
+	//		)
+	//		return mconfigToUpdate(res.Configs, res.LogicalID, receivedDigest.Md5HexDigest)
+	//	}
+	//}
+	//
```

Comment:
Delete all the commented code. You can't predict if these commented sections will still work when we get around to this. The code will still be there in vcs anyways